test/analyzers/lint_analyzer/rules/rules_list/prefer_const_border_radius/prefer_const_border_radius_test.dart

```diff
@@ -27,33 +27,41 @@ void main() {
 
         RuleTestHelper.verifyIssues(
           issues: issues,
-          startOffsets: [64, 170, 269, 417],
-          startLines: [2, 5, 8, 12],
-          startColumns: [22, 31, 31, 31],
-          endOffsets: [88, 195, 295, 442],
+          startOffsets: [113, 273, 373, 440, 588, 870],
+          startLines: [4, 9, 12, 13, 17, 22],
```

Comment:
I see seven `// LINT` comments and only 6 actual triggers (line 25 doesn't trigger the rule). Please update the test file.